### ISSUE TITLE
Translations es_ES or Es_MX

### ISSUE THREAD

**Issue by PanuWeb** [espocrm/espocrm#712]
Hello

I consult them because I usually collaborate with translations into Spanish. I live in Argentina and now from version 4.8 EspoCRM supports the Mexican variation of Spanish (es_MX).

What do you prefer to continue with the Spanish translation of Spain (en_ES) or directly do my contributions for the Mexican variation (es_MX) with which I am more familiar?

In fact es_MX has several translations to improve

Regards!

**Comment by yurikuzn** [comment]
Hi Mauricio,

Both would be fine. BTW we have an account on POEditor https://poeditor.com/join/project/gLDKZtUF4i where it could be much easier to contribute. When I see changes on POEditor for some language I export-import changes into the repository. 

Thank for your help

**Comment by PanuWeb** [comment]
Thanks for the reply.

At the moment I am going to focus on the Spanish translation of Spain. I can not do it with both (I do not have time)

I already created an account in POEditor and uploaded my last .PO file to the translations es_ES, many translations were missing, translations that I already had ready and had shared on GitHub.

In conclusion now we go from 80% translated to 94% in es_ES. This weekend I will complete the missing translations.

I recently discovered your YouTube channel, and it was very useful for some areas that I did not have clear.

I love the work you are doing with CRM!

Regards!
